src/foundry/prosemirror/keymaps.d.mts

```diff
@@ -30,15 +30,14 @@ declare class ProseMirrorKeyMaps extends ProseMirrorPlugin {
    * @param schema  - The ProseMirror schema to build keymaps for.
    * @param options - Additional options to configure the plugin's behaviour.
    */
-  constructor(schema: Schema, options?: ProseMirrorKeyMaps.Options);
+  constructor(schema: Schema, { onSave }?: ProseMirrorKeyMaps.Options);
```

Comment:
Keep it as the `options` so the parameter annotations match.

Reply:
This is just me forgetting to replace it with the `onSave` parameter annotation from source. No other options are used - can I keep it updated but just change the annotation to be for `onSave`?

Reply:
I *think* the type annotations won't carry through properly if you do that. This is another unfortunate example of we need to interpolate better types than what Foundry offers.

Reply:
Gotcha. Lemme check and make sure I haven't accidentally done that anywhere else